zpay32/invoice_test.go

```diff
@@ -805,25 +889,42 @@ func TestNewInvoice(t *testing.T) {
 			valid:          true,
 			encodedInvoice: "lnbcrt241pvjluezpp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdqqnp4q0n326hr8v9zprg8gsvezcch06gfaqqhde2aj730yg0durunfhv66df5c8pqjjt4z4ymmuaxfx8eh5v7hmzs3wrfas8m2sz5qz56rw2lxy8mmgm4xln0ha26qkw6u3vhu22pss2udugr9g74c3x20slpcqjgq0el4h6",
 		},
+		{
+			// Mainnet invoice with two blinded paths.
```

Comment:
well it would be part of the bolt11 addition right?

Reply:
ahh ok, so we are going to not write a Blip for this but enhance the BOLT11 spec via a PR ? Sounds good as well

Reply:
yeah I think initially let's start with a bLIP since this is a "fringe-only" suggestion guarded by an experimental feature bit _and_ cause it is not the intended way to communicate a blinded path. If this does happen to become more widely used, then we can add it to the BOLTs

Reply:
added test vector to blip!